FAQ - Add section about domain verification (#2037)
vercel/docs: pages/docs/v2/platform/frequently-asked-questions.mdx

These steps are required for every URL and cookie values cannot be shared across Deployments.

## Domain Verification via CLI

> It's not required to manually verify domains with TXT records anymore.

Reviewer: ```suggestion
It is no longer required to manually verify domains via TXT records anymore.
```

> Instead, once a domain has been added to a Project, its DNS settings have to be adjusted properly. More information about the configuration can be found in the <Link href="/docs/v2/custom-domains#step-4:-configuring-the-domain">Custom Domains</Link> section.

Reviewer: ```suggestion
Instead, a domain's DNS settings must to be configured after the domain is added to a Project. See the <Link href="/docs/v2/custom-domains#step-4:-configuring-the-domain">Custom Domains</Link> section for more information.
```
